lib/LightPanels/LightPanels.js

```diff
@@ -51,10 +51,26 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	indexChanged: function (previousIndex) {
+	animateTo: function (index) {
+		this.updateSpottability(this.index, index);
+		LightPanels.prototype.animateTo.apply(this, arguments);
+	},
+
+	/**
+	* @private
+	*/
+	indexChanged: function () {
+		this.updateSpottability.apply(this, arguments);
```

Comment:
While effectively identical, I think I'd prefer specifying the arguments on `indexChanged` and explicitly passing them to `updateSpottability` rather than apply. I think it's a bit clearer.